mrp_production_partner_note/models/procurement_order.py

```diff
@@ -0,0 +1,19 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, api
+
+
+class ProcurementOrder(models.Model):
+    _inherit = 'procurement.order'
+
+    @api.model
+    def _prepare_mo_vals(self, procurement):
+        res = super(ProcurementOrder, self)._prepare_mo_vals(procurement)
+        sale_proc = procurement.move_dest_id.procurement_id
+        mrp_notes = sale_proc.sale_line_id.order_id.partner_id.mrp_notes
+        if mrp_notes:
+            res['notes'] = mrp_notes
```

Comment:
Here you are overwritting.

It should be an append dont you think?

Something like:

res['notes'] = '%s\n\n---\n\n%s' % (res.get('notes'), notes)

With that you ensure if any other module is creating notes you will not lose them.